akka-stream-tests-tck/src/test/scala/akka/stream/tck/Timeouts.scala

```diff
@@ -12,8 +12,8 @@ import akka.testkit._
  */
 object Timeouts {
 
-  def publisherShutdownTimeoutMillis: Int = 1000
+  def publisherShutdownTimeoutMillis: Int = 3000
```

Comment:
there's a few "expect no message" but if it's not too painful then sure, better than racy tests :)